README.md

```diff
@@ -25,53 +25,157 @@
 
 ---
 
-REAFLOW is a modular diagram engine for build static or interactive editors.
-The library is feature rich and modular allowing for displaying complex visualizations
-with total customizability.
+REAFLOW is a modular diagram engine for building static or interactive editors. The library is feature-rich and modular allowing for displaying complex
+visualizations with total customizability.
 
 ## 🚀 Quick Links
-- Checkout the [docs and demos](https://reaflow.dev)
-- Learn about updates from the [Changelog](CHANGELOG.md)
+
+- :point_right: Checkout the [**docs and demos**](https://reaflow.dev)
+- :newspaper: Learn about updates from the [Changelog](CHANGELOG.md)
 - Explore the library on [Chroma](https://www.chromatic.com/library?appId=5f99ba42fe88ac0022fd1147)
 
 ## 📦 Usage
-Install the package via NPM:
+
+Install the package via **NPM**:
 
 ```
 npm i reaflow --save
 ```
 
-then add the component with some nodes and edges:
+Install the package via **Yarn**:
+
+```
+yarn add reaflow
+```
+
+## Usage with Create React App (no SSR support, client-side support only)
+
+Add the component with some nodes and edges:
 
 ```jsx
 import React from 'react';
 import { Canvas } from 'reaflow';
 
 export default () => (
-  <Canvas
-    nodes={[
-      {
-        id: '1',
-        text: '1'
-      },
-      {
-        id: '2',
-        text: '2'
-      }
-    ]}
-    edges={[
-      {
-        id: '1-2',
-        from: '1',
-        to: '2'
-      }
-    ]}
-  />
+  <div style={{ position: 'relative', width: '100vw', height: '100vh' }}>
+    <div style={{ position: 'absolute', top: 0, bottom: 0, left: 0, right: 0, 'background-color': 'lightgrey' }}>
+      <Canvas
+        maxWidth={800} // Use small canvas dimensions to make sure the nodes appear on screen immediately
```

Comment:
The comment would cause this to not work.

Reply:
I believe those comments are valid. I use them in my own code on many other projects.

![image](https://user-images.githubusercontent.com/3807458/109550808-0de79880-7ad0-11eb-9dbc-76cf9881aa04.png)

Reply:
Really?! I swore that wouldn't work - thats OK then.